intelmq/bots/BOTS

```diff
@@ -598,6 +605,17 @@
                 "hierarchical_output": false
             }
         },
+        "Files": {
+            "description": "Files is the bot responsible for lockfree saving of events into separate files.",
+            "module": "intelmq.bots.outputs.files.output",
+            "parameters": {
+                "tmp": "/opt/intelmq/var/lib/bots/files-output/tmp",
+                "dir": "/opt/intelmq/var/lib/bots/files-output/incoming",
+                "suffix": ".json",
+                "hierarchical_output": false,
+                "single_key": "output"
```

Comment:
Hmm, even I think you're right, I was trying to be consistent with 'file' bot in hierarchical_output setting, but I haven't noticed this. I'll update to null, so output defaults to the whole event.